src/main/java/teammates/ui/controller/InstructorStudentListPageCourseData.java

```diff
@@ -0,0 +1,25 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.CourseAttributes;
+
+/**
+ * Serves as a datatransfer class between {@link InstructorStudentListPageAction}
+ * and {@link InstructorStudentListPageData}
+ */
+public class InstructorStudentListPageCourseData {
```

Comment:
Well yeah, so that `ISLPageData` is freed from logic. Maybe it's not in the right package for now (maybe this is better off in datatransfer?), but I'm going to need this class.